src/locales/ar-SA.json

```diff
@@ -19,26 +19,27 @@
   "strategiesPage": "الإستراتيجية (تخطيط)",
   "space": "المساحة",
   "spaces": "المساحات",
+  "verifiedSpace": "Verified space",
   "version": "الاصدار",
   "timeline": "الجدول الزمني",
   "filters": "المرشحات",
   "allSpaces": "جميع المساحات",
   "submitOnchain": "إرسال على السلسلة",
   "inSpaces": "في {0} مساحة(مساحات)",
-  "downloadReport": "تنزيل التقرير",
   "votes": "الأصوات",
   "seeMore": "الاطّلاع على المزيد",
   "network": "الشبكة",
   "networks": "الشبكات",
   "skins": "المظاهر",
-  "members": "No members | {count} member | {count} members",
+  "members": "لا يوجد أعضاء <unk> {count} أعضاء <unk> {count} أعضاء",
```

Comment:
Something wrong here @metrox-eth

Reply:
@metrox-eth Can you fix this and check other text if its fine then approve the pull request so we can merge it?

Reply:
Fixed in Crowdin, but closed this PR to fix conflicts